nilearn/connectome/connectivity_matrices.py

```diff
@@ -193,9 +193,10 @@ def _geometric_mean(matrices, init=None, max_iter=10, tol=1e-7):
     return gmean
 
 
-def sym_to_vec(symmetric):
-    """Return the flattened lower triangular part of an array, after
-    multiplying above the diagonal elements by sqrt(2).
+def sym_to_vec(symmetric, discard_diagonal=False, isometric=True):
+    """Return the flattened lower triangular part of an array.
+
+    If diagonal is kept, off-diagonal elements are multiplied by sqrt(2).
```

Comment:
I thought we had agreed on a different behavior: when the diagonal is kept, it is scaled by 1. / sqrt(2). This might be less confusing to users.